### DISCUSSION TITLE
Need help with changing email template for verification

### DISCUSSION THREAD

**Discussion by TheLazyDev** [ory/kratos#4160] [Q&A] [answered]
I have a self hosted kratos using docker and I am unable to change account verification email template. I tried to follow the documentation but I am unsuccessful. In the guide it's written as below:

```
When self-hosting Ory Kratos, the system expects the templates at these paths:

<Kartos-root>/<templates-root>/recovery


```
I am unable to follow through so please help

**Comment by vinckr** [comment, marked as answer]
Hello @TheLazyDev 
you need to add a bit more information as to what you problem is.
Take a look at this blogpost as well: https://www.ory.sh/custom-email-templates/ - its a bit dated but should still be useful for your issue.
